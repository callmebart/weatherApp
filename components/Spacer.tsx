import { View } from 'react-native'
interface Props {
    spacingValue?: number;
}
export default function Spacer({ spacingValue = 1 }: Props) {
    return <View style={{ height: 8 * spacingValue }} />;
}