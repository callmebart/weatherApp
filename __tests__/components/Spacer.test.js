import { render } from "@testing-library/react-native";
import Spacer from "../../components/Spacer";

describe('Spacer', () => {
  it('should render a View with a style that has a height of 8 times the spacingValue prop', () => {
    const root  = render(<Spacer spacingValue={2} />).toJSON();
    expect(root).toMatchSnapshot();
  });
});
