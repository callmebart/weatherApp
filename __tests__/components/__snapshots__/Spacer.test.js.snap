// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer should render a View with a style that has a height of 8 times the spacingValue prop 1`] = `
<View
  style={
    Object {
      "height": 16,
    }
  }
/>
`;
